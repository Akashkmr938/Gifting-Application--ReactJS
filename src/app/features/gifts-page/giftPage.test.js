import React from "react";
import renderer from "react-test-renderer";
import GiftPage from "./giftspage";

describe("The GiftPage component", () => {
  const props = {
    location: {
      search: ""
    }
  };
  it("renders correctly", () => {
    const tree = renderer.create(<GiftPage {...props}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
