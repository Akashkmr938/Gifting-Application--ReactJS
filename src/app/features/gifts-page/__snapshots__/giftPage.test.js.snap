// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The GiftPage component renders correctly 1`] = `
<h3>
  No Data To Display
</h3>
`;
